uint8 TYPE_VELOCITY=0
uint8 TYPE_POSITION=1
uint8 TYPE_VELDELTA=2
uint8 TYPE_TRAJECTORY=3
uint8 TYPE_TRAJECTORY_LOOP=4
uint8 TYPE_TRAJECTORY_RESET=5

string object_id       # Id of object to control
uint8 controller_type  # Type of controller to instantiate
---
bool success           # Was the call successful
string error_msg       # Error message, if any